tutorials/cp-cf-sapui5-local-debug/cp-cf-sapui5-local-debug.md

```diff
@@ -3,8 +3,8 @@ title: Debug Your SAPUI5 App
 description: Debugging and logging are the nut and bolts to inspect your application. Learn how to debug and test your web apps with the Google Chrome Dev Tools.
 auto_validation: true
 time: 15
-tags: [ tutorial>beginner, topic>html5, topic>cloud, topic>javascript, products>sap-cloud-platform-for-the-cloud-foundry-environment, tutorial>free-tier]
-primary_tag: topic>sapui5
+tags: [ tutorial>beginner, programming-tool>html5, topic>cloud, programming-tool>javascript, products>sap-cloud-platform-for-the-cloud-foundry-environment, tutorial>free-tier]
```

Comment:
Please update tags. products>sap-cloud-platform-for-the-cloud-foundry-environment -> products>sap-btp--cloud-foundry-environment

Reply:
done, thanks a lot